config/development.json

```diff
@@ -153,6 +153,7 @@
 		"signup/design-picker-premium-themes-checkout": true,
 		"signup/design-picker-unified": true,
 		"signup/design-picker-use-featured-picks-buttons": true,
+		"signup/build-your-own": true,
```

Comment:
@bangank36 can we "prefix" this flag with the text `design-picker` (`signup/design-picker-build-your-own`)?  And also, this section is sorted alphabetically!